README.md

```diff
@@ -2,3 +2,59 @@
 Practice repo for people to try things and learn to use git and github
 
 Here I made a change that will cause another branch to be out of date for the PR
+
+Here's a nice poem:
+
+```
+Romans in Dorset: A.D. MDCCCXCV
+BY LOUISE IMOGEN GUINEY
+A stupor on the heath,
```

Comment:
Maybe leave one blank line before the poem starts?

Reply:
hey, thanks, this has been added